rebirthdb/ast.py

```diff
@@ -852,75 +851,74 @@ def compose(self, args, optargs):
 
 
 class MakeArray(RqlQuery):
-    tt = pTerm.MAKE_ARRAY
+    term_type = P_TERM.MAKE_ARRAY
 
     def compose(self, args, optargs):
         return T('[', T(*args, intsp=', '), ']')
 
 
 class MakeObj(RqlQuery):
-    tt = pTerm.MAKE_OBJ
+    term_type = P_TERM.MAKE_OBJ
 
     # We cannot inherit from RqlQuery because of potential conflicts with
     # the `self` parameter. This is not a problem for other RqlQuery sub-
     # classes unless we add a 'self' optional argument to one of them.
+    # TODO: @gabor-boros Figure out is the above still an issue or not.
```

Comment:
This is a permanent issue. See RebirthDB/rebirthdb-python@b62a006a13035920ac4238731f22410fa21d60f5 for why. The real problem is at call sites. All other queries use Python's kwargs syntax to pass options.